src/tasks/minions/maniacalMonkeyActivity.ts

```diff
@@ -0,0 +1,73 @@
+import { Bank } from 'oldschooljs';
+import { SkillsEnum } from 'oldschooljs/dist/constants';
+
+import { resolveAttackStyles } from '../../lib/minions/functions';
+import { handleTripFinish } from '../../lib/util/handleTripFinish';
+import { makeBankImage } from '../../lib/util/makeBankImage';
+import { ManiacalMonkeyTable } from './../../lib/simulation/maniacalMonkey';
+import { ManiacalMonkeyTaskOptions } from './../../lib/types/minions';
+
+export const maniacalMonkeyID = 6803;
+
+export const maniacalMonkeyTask: MinionTask = {
+	type: 'ManiacalMonkey',
+	async run(data: ManiacalMonkeyTaskOptions) {
+		const { userID, channelID, quantity, duration } = data;
+		const user = await mUserFetch(userID);
+
+		const [, , attackStyles] = resolveAttackStyles(user, { monsterID: maniacalMonkeyID });
+
+		const manicalMonkeyHP = 65;
+		const totalXP = manicalMonkeyHP * 4 * quantity;
+		const xpPerSkill = totalXP / attackStyles.length;
+
+		let res: string[] = [];
+
+		for (const style of attackStyles) {
+			res.push(
+				await user.addXP({
+					skillName: style,
+					amount: Math.floor(xpPerSkill),
+					duration
+				})
+			);
+		}
+
+		res.push(
+			await user.addXP({
+				skillName: SkillsEnum.Hitpoints,
+				amount: Math.floor(manicalMonkeyHP * quantity * 1.33),
+				duration
+			})
+		);
+
+		let loot = new Bank();
+
+		for (let i = 0; i < quantity; i++) {
+			loot.add(ManiacalMonkeyTable.roll());
+		}
```

Comment:
You're not giving all this loot are you? Nobody gets every single loot from monkeys banked after killing them. Only stackable things?